source/bin/tvm.rs

```diff
@@ -154,10 +166,8 @@ mod tests {
     #[test]
     fn case_usage() {
         assert_eq!(
-            "Usage: tvm [options] [file]\n".to_string() +
-            "Options:\n" +
-            "    -v, --version    Print version.\n" +
-            "    -h, --help       This help.",
+            "Usage: tvm [options] [file]\nOptions:\n    -v, --version    \
```

Comment:
I know, but I saw it and thought you might want to do something further with that and make it a serious test so I thought doing that might be a good idea. Change to whatever you want.